utils/containers/sets/hash_set_generator.bzl

```diff
@@ -0,0 +1,26 @@
+def hash_set_generate(size):
```

Comment:
Instead of relying on sed, could you use 
an approach similar to https://github.com/bazelbuild/examples/blob/master/rules/expand_template/hello.bzl so that we're OS-agnostic? docs are at https://docs.bazel.build/versions/master/skylark/lib/actions.html#expand_template